### PR TITLE
link default

### PR DESCRIPTION
**Pull request by xsrobin**

### CODE REVIEW

File: doc/fluid/user_guides/models/index_cn.rst [hunk 1]
```diff
@@ -184,7 +184,7 @@ SimNet是百度自然语言处理部于2013年自主研发的语义匹配框架
 
 百度阅读理解数据集是由百度自然语言处理部开源的一个真实世界数据集，所有的问题、原文都来源于实际数据(百度搜索引擎数据和百度知道问答社区)，答案是由人类回答的。每个问题都对应多个答案，数据集包含200k问题、1000k原文和420k答案，是目前最大的中文MRC数据集。百度同时开源了对应的阅读理解模型，称为DuReader，采用当前通用的网络分层结构，通过双向attention机制捕捉问题和原文之间的交互关系，生成query-aware的原文表示，最终基于query-aware的原文表示通过point network预测答案范围。
 
--  `DuReader in PaddlePaddle Fluid <https://github.com/PaddlePaddle/models/blob/develop/fluid/PaddleNLP/machine_reading_comprehension/README.md>`__
+-  `DuReader in PaddlePaddle Fluid <https://github.com/PaddlePaddle/models/blob/develop/PaddleNLP/machine_reading_comprehension/README.md>`__
```
Comment (acosta123) [inline doc/fluid/user_guides/models/index_cn.rst:187]: 正确链接应为https://github.com/PaddlePaddle/models/blob/develop/PaddleNLP/unarchived/machine_reading_comprehension/README.md

File: doc/fluid/user_guides/models/index_cn.rst [hunk 2]
```diff
@@ -163,7 +163,7 @@ DQN 及其变体，并测试了它们在 Atari 游戏中的表现。
 
 本例所开放的DAM (Deep Attention Matching Network)为百度自然语言处理部发表于ACL-2018的工作，用于检索式聊天机器人多轮对话中应答的选择。DAM受Transformer的启发，其网络结构完全基于注意力(attention)机制，利用栈式的self-attention结构分别学习不同粒度下应答和语境的语义表示，然后利用cross-attention获取应答与语境之间的相关性，在两个大规模多轮对话数据集上的表现均好于其它模型。
 
-- `Deep Attention Matching Network <https://github.com/PaddlePaddle/models/tree/develop/fluid/PaddleNLP/deep_attention_matching_net>`__
+- `Deep Attention Matching Network <https://github.com/PaddlePaddle/models/tree/develop/PaddleNLP/deep_attention_matching_net>`__
```
Comment (acosta123) [inline doc/fluid/user_guides/models/index_cn.rst:166]: 正确链接应为
https://github.com/PaddlePaddle/models/blob/develop/PaddleNLP/unarchived/deep_attention_matching_net/README.md

**Review by acosta123 — CHANGES_REQUESTED** [review]
有的链接放错了，建议修改

**Comment by shanyi15** [comment]
close via https://github.com/PaddlePaddle/FluidDoc/pull/876

**Pull request closed without merging**